chain/x/zoracle/internal/types/msgs.go

```diff
@@ -62,6 +65,9 @@ func (msg MsgRequestData) ValidateBasic() sdk.Error {
 	if msg.Expiration <= 0 {
 		return sdk.ErrUnknownRequest("Expiration period must be greater than zero")
 	}
+	if msg.PrepareGas == 0 {
```

Comment:
Let's use <= comparison here and below, so we don't fuck up if we somehow decide to change variable type to int64